app/controllers/search_controller.rb

```diff
@@ -3,19 +3,55 @@ class SearchController < ApplicationController
 
   def index
     unless params[:query].empty?
-      track_action
-
       respond_to do |format|
-        format.js { redirect_to build_filter_path(:search, params[:query].gsub(".", "")) }
-        format.html { redirect_to build_filter_path(:search, params[:query].gsub(".", "")) }
+        format.js { redirect_to build_filter_path(:search, params[:query]) }
+        format.html { redirect_to build_filter_path(:search, params[:query]) }
       end
     else
       redirect_back fallback_location: root_path
     end
   end
 
+  def old_index
```

Comment:
Maybe we could come up with a better name. `redirect_to_param` or something, not entirely sure. Or add a comment what this is.